exotic.py

```diff
@@ -1330,31 +1384,43 @@ def parse_args():
             date = str(input("\nEnter the Observation Date: "))
 
         if fitsortext == 1:
-            # latitude and longitude
             if fileorcommandline == 1:
-                latiStr = str(input("Enter the latitude of where you observed (deg) (Don't forget the sign where North is '+' and South is '-'): "))
-            noSpaceLati = latiStr.replace(" ", "")
-            latiSign = noSpaceLati[0]
-            # check to make sure they have a sign
-            while latiSign != '+' and latiSign != '-':
-                print("You forgot the sign for the latitude! North is '+' and South is '-'. Please try again.")
-                latiStr = str(input("Enter the latitude of where you observed (deg) (Don't forget the sign where North is '+' and South is '-'): "))
-                noSpaceLati = latiStr.replace(" ", "")
-                latiSign = noSpaceLati[0]
-            lati = float(latiStr)
-
-            # handle longitude
+                latiStr = input("Enter the latitude of where you observed (deg) "
+                                "(Don't forget the sign where North is '+' and South is '-'): ")
+            # Latitude
+            while True:
```

Comment:
just a question- doesn't this while loop always run since it is always true? If so, why use this coding? (just curious!)

Reply:
There's a break at the end. If it passes all my parameters, it will break. If it doesn't, it will raise an error and loop over again.